app/models/user.rb

```diff
@@ -11,5 +11,6 @@ class User < ActiveRecord::Base
     has_secure_password
     has_many :items, dependent: :destroy
     has_many :reviews, dependent: :destroy
+    has_many :receipts, dependent: :destroy
```

Comment:
This dependency is too strong. A receipt should only be destroyed if both the buyer and the seller and deleted, not one or the other. We don't want sellers having their receipt history deleted.